src/ameba/ast/scope.cr

```diff
@@ -195,6 +195,16 @@ module Ameba::AST
       outer_scope.nil?
     end
 
+    def inherits?
```

Comment:
This reads better, as adjective demands a following noun, which there's none in this case.

```suggestion
    def inherited?
```